### DISCUSSION TITLE
RfC - Solution for Slack messages disappearing after 90 days

### DISCUSSION THREAD

**Discussion by macolo** [django-cms/django-cms#7836] [General]
## Intro
- Please read this first: [Why tech groups can use Slack for free but open source projects and businesses of any size shouldn’t](https://medium.com/@gabriel_wilkes/why-tech-groups-can-get-away-with-using-slack-for-free-but-open-source-projects-and-businesses-of-96c427aaefbb)
- We have the problem that messages are lost after 90 days for two use cases:
   - knowledge in support channels should be browseable
   - decisions made by the leadership team and the dCA board should be documented

## Solution Approaches
- We are exploring different possibilities:
   - switching to Discord ([Sign up here](https://discord-main-channel.django-cms.org)), technical questions should go to stackoverflow
   - using https://github.com/felixrieseberg/slack-archive for:
      - channels where decisions are made (#dca-leadership-team, #dca-admin, #dca-board, #technical-committee, #*-fellow, #workgroup-security - those exports could remain in a private github repository
      - the #support channel, which could remain in a public github repository (published to github pages)

**Comment by marksweb** [comment]
For visibility, we are back on Discord; https://discord.gg/xkQ2tbKy

And some people do take their tech questions to Stack Overflow; https://stackoverflow.com/questions/tagged/django-cms?sort=RecentActivity

**Comment by macolo** [comment]
Were switching to Discord as per votes of the feet.
